### PR TITLE
homework

### PR DESCRIPTION
**Pull request by jimhantrix**
Jim Tekanyo 
"still have problems using CL(command-line) and git& github"

### CODE REVIEW

File: functions.js [hunk 1]
```diff
@@ -1 +1,102 @@
...
+
+repeatPhrase("Hello", 5);
+
+
+//3.power function  (return)
+function toTheNthPower(number,power){
+    var result = 1
+    for (var i = 0; i < power ,i ++);
+}
+
```
Comment (Nick-Brennan) [inline functions.js]: You are missing a code block after you instantiate your for loop.
for(var i = 0; i < power ; i++){
  result *= number
}

File: functions.js [hunk 2]
```diff
@@ -1 +1,102 @@
...
+
+//4. Area of a circle  (return)
+function areaOfACircle(radius){
+ var i ===3.14
+  var result ( i * radius * 2 )
+}
+areaOfACircle(2);
+return  result; 12.57
+
```
Comment (Nick-Brennan) [inline functions.js]: i \* radius \* radius

File: functions.js [hunk 3]
```diff
@@ -1 +1,102 @@
...
+
+//5. Pythagorean theorema(return)
+function  pythagoreanTheorem( a, b){
+var result( a + b)*2)===c);
+}
+pythagoreanTheorem (3,4);
+return ; true
+
```
Comment (Nick-Brennan) [inline functions.js:25]: We should go over this together

**Comment by jimhantrix** [comment]
Still trying to understand the syntax ,I think i need to review last sessions not yet fully aware of the flow

**Pull request closed without merging**